src/clr-angular/data/datagrid/datagrid-row.ts

```diff
@@ -260,4 +242,58 @@ export class ClrDatagridRow<T = any> implements AfterContentInit, AfterViewInit
   public get _view() {
     return this.wrappedInjector.get(WrappedRow, this.vcr).rowView;
   }
+
+  private resetViewsOnDisplayModeChange(): Subscription {
```

Comment:
nit: naming seems to reflect resetting back to some base state. it seems like its just inserting views after display mode changes.
suggestions: insertViewsAfterDisplayModeChange, insertViewsAfterModeChange